pdns/lua-record.cc

```diff
@@ -1061,16 +1061,20 @@ static void setupLuaRecords(LuaContext& lua) // NOLINT(readability-function-cogn
       ComboAddress ca(address);
```

Comment:
If you add an explicit return type (` -> vector<string>`, you can get rid off all the `vector<string>` in the body, i.e. use `return {address}` or `return {}`.

Reply:
oh i love it

Reply:
done